src/query_parser/v2/lexer.rl

```diff
@@ -29,9 +29,9 @@ void RSQuery_ParseFree_v2(void *p, void (*freeProc)(void *));
 
 machine query;
 
-inf = ['+\-']? 'inf' $ 4;
+inf = ('+'|'-')? 'inf'i $ 4;
 size = digit+ $ 2;
-number = '-'? digit+('.' digit+)? (('E'|'e') '-'? digit+)? $ 3;
+number = ('+'|'-')? digit+('.' digit+)? (('E'|'e') ('+'|'-')? digit+)? $ 3;
```

Comment:
Same here, can use `[+\-]`.
Doesn't matter at all but I guess it's nice to use the specific syntax of the lexer.